packages/catalog-readmodel-writer/test/consumerService.integration.test.ts

```diff
@@ -106,6 +109,7 @@ describe("database test", async () => {
         type: "EServiceAdded",
         event_version: 1,
         data: payload,
+        log_date: new Date(),
```

Comment:
Super minor, maybe check if there are some other `.integration.test.ts` files where you now need to add `log_date`, TS build does not complain and tests still work but you should see errors in VSCode in case there are

Reply:
as discuss offline `EventEnvelope` type are used only by `consumerService.integration.test.ts`

Reply:
I think it's used in all readmodel-writer tests because it is the generic type used to define all the event envelopes such as `AttributeEventEnvelope` or `AgreementEventEnvelope` used in the respective tests (here https://github.com/pagopa/interop-be-monorepo/blob/7030e8e0b231f6e3e2249ad3a31c2061f3d69cb6/packages/agreement-readmodel-writer/test/agreement-readmodel-writer.integration.test.ts#L14 or here https://github.com/pagopa/interop-be-monorepo/blob/7030e8e0b231f6e3e2249ad3a31c2061f3d69cb6/packages/attribute-registry-readmodel-writer/test/attribute-readmodel-writer.integration.test.ts#L17)

Reply:
my fault wrong search 👍🏻